CausalEstimate/estimators/functional/variance.py

```
@@ -0,0 +1,108 @@
+        ci_lower = psi - 1.96 * std_err
+        ci_upper = psi + 1.96 * std_err
+
+    return {STD_ERR: std_err, CI95_LOWER: ci_lower, CI95_UPPER: ci_upper}
+
+
+def _compute_ic_ate(
+    psi: float,
+    Q_star_1: np.ndarray,
+    Q_star_0: np.ndarray,
+    Y: np.ndarray,
+    A: np.ndarray,
```

Ruff (0.13.3)

Unused function argument: `A`

(ARG001)